src/Microsoft.AspNet.Razor/Generator/Compiler/CodeBuilder/CSharp/CSharpTagHelperCodeRenderer.cs

```diff
@@ -345,23 +351,28 @@ private void RenderUnboundHTMLAttributes(IEnumerable<KeyValuePair<string, Chunk>
                     .WriteStartInstanceMethodInvocation(
                         ExecutionContextVariableName,
                         _tagHelperContext.ExecutionContextAddHtmlAttributeMethodName)
-                    .WriteStringLiteral(htmlAttribute.Key)
-                    .WriteParameterSeparator()
-                    .WriteStartMethodInvocation(_tagHelperContext.MarkAsHtmlEncodedMethodName);
+                    .WriteStringLiteral(htmlAttribute.Key);
 
-                // If it's a plain text value then we need to surround the value with quotes.
-                if (isPlainTextValue)
+                // If we have a minimized attribute we don't want to provide a value
```

Comment:
agree with describing why but "don't want" provides no information. there simply _is_ no value when `attributeValue == null`.  could repeat something about a minimized attribute (as on line 331) but that seems redundant.